From b09fcc2b2826a74cab076ae3280f7bc928507de5 Mon Sep 17 00:00:00 2001
From: Edwin Moquete <edwinmmoquete@gmail.com>
Date: Wed, 4 Oct 2023 02:28:26 +0530
Subject: [PATCH] pdx206: Remove unused persist.bluetooth.a2dp_offload.cap
 property

This prop has no effect as of T QPR1.

See: https://github.com/LineageOS/android_packages_modules_Bluetooth/commit/cf25e331479a850ad888f077b1d6bac7419213e4

Change-Id: Ic762812dd59429d344ccc55c01bf96b0ffd6dbab
---
 vendor.prop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vendor.prop b/vendor.prop
index 0ab372d..cc0caaf 100644
--- a/vendor.prop
+++ b/vendor.prop
@@ -128,7 +128,6 @@ bluetooth.profile.sap.server.enabled?=true
 bluetooth.profile.ccp.server.enabled?=true
 bluetooth.profile.vcp.controller.enabled?=true
 persist.bluetooth.a2dp_offload.disabled=false
-persist.bluetooth.a2dp_offload.cap=sbc-aac-aptx-aptxhd-ldac
 persist.vendor.bt.a2dp_offload_cap=sbc-aptx-aptxtws-aptxhd-aac-ldac
 persist.vendor.bt.aac_frm_ctl.enabled=true
 persist.vendor.bt.aac_vbr_frm_ctl.enabled=true
